internal/log/stdLogger.go

```diff
@@ -0,0 +1,51 @@
+package log
+
+import (
+	"fmt"
+	"io"
+	goLog "log"
```

Comment:
No need to do this. Just leave it as log.

Reply:
Inside a "log" package, I think this makes references more clear.